docs/reference/changelog.md

```diff
@@ -114,11 +115,13 @@
     * 'transport.lowCarbon'
     * 'naturalResources'
     * 'naturalResources.floodProtection'
+  * [#455](https://github.com/open-contracting/infrastructure/pull/455) - locationGazetteers: 'WDPA'
   * documentType:
     * [#449](https://github.com/open-contracting/infrastructure/pull/449) - 'materialTestResults'
     * [#457](https://github.com/open-contracting/infrastructure/pull/457) - 'climateMeasures'
 * [#432](https://github.com/open-contracting/infrastructure/pull/432) - Deprecate 'completed' in favor of 'maintenance' in the projectStatus codelist
 
+
```

Comment:
Remove extra newline

Reply:
Done in https://github.com/open-contracting/infrastructure/pull/455/commits/1de13bfad42ce2b55250579a38a4bc106607412a